mathjax3-ts/core/InputJax.ts

```diff
@@ -63,11 +64,26 @@ export interface InputJax<N, T, D> {
      */
     adaptor: DOMAdaptor<N, T, D>;
 
+    /**
+     * The MmlFactory for this input jax
+     */
+    mmlFactory: MmlFactory;
```

Comment:
Do we ever need a getter for this?

Reply:
Not from outside the InputJax.  We need a setter because the MathDocument will pass its factory on to the input and output jax, but there is no need to get the factory from outside (the document already knows what it is).  Same for adaptors.